shared/draft-utils/test/__snapshots__/add-embeds-to-draft-js.test.js.snap

```diff
@@ -0,0 +1,43 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should add embeds 1`] = `
+Object {
+  "blocks": Array [
+    Object {
+      "data": Object {},
+      "depth": 0,
+      "entityRanges": Array [],
+      "inlineStyleRanges": Array [],
+      "key": "g0000",
+      "text": "https://simplecast.com/s/a1f11d11",
+      "type": "unstyled",
+    },
+    Object {
+      "data": Object {},
+      "depth": 0,
+      "entityRanges": Array [
+        Object {
+          "key": -Infinity,
+          "length": 1,
+          "offset": 0,
+        },
+      ],
+      "inlineStyleRanges": Array [],
+      "key": "0",
+      "text": " ",
+      "type": "atomic",
+    },
+  ],
+  "entityMap": Object {
+    "-Infinity": Object {
```

Comment:
Is this `-Infinity` intentional? Or should that be an actual key number?

Reply:
Yes that should be 0, I fixed that bug in #4827 or #4772 or some other of my recent PRs. Let's merge those first and then update this test?

Reply:
Merged #4772 - let's make sure the snapshot test fails and then we can update it, otherwise we know the fix is in #4827

Reply:
Hm, just flow failures now @mxstbr 🤔